test/rantly_generator_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+require 'rantly/minitest_extensions'
+
+describe Rantly do
+  before do
+    Rantly.gen.reset
+  end
+
+  describe '#range' do
+    it 'returns an integer within range when given two integers' do
+      property_of {
+        i = integer
+        [i,range(i,i)]
+      }.check { |(a,b)|
```

Comment:
[nitpick] what about given `a` and `b` more explicit names? For example, `i`, `range_i`

Reply:
PR updated.